actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Allow variants to be set through the URL
+
+    request.variant can be set by setting :format. For example, accessing
+    http://example.com/posts.html+partial or `request.format = 'html+partial'` set
```

Comment:
Mentioning partial here is confusing (both format and variant affect which partial is rendered), switch to using phone.